modules/content.js

```diff
@@ -0,0 +1,35 @@
+const home = document.getElementById('home__li');
+const addBook = document.getElementById('add__book__li');
+const contact = document.getElementById('contact__li');
+
+// Show list view
+function showlist() {
+  home.addEventListener('click', () => {
+    document.querySelector('.list-container').style.display = 'flex';
+    document.querySelector('#add__book').style.display = 'none';
+    document.querySelector('#contact').style.display = 'none';
+  });
+}
+
+// Show add book view
+function showAdd() {
+  addBook.addEventListener('click', () => {
+    document.querySelector('#home').style.display = 'none';
+    const newBookContainer = document.querySelector('.new-book-container');
+    if (newBookContainer) {
+      newBookContainer.style.display = 'flex';
+    }
+    document.querySelector('#contact').style.display = 'none';
+  });
+}
+
+// Show contact view
+function showContact() {
```

Comment:
- [ ] As a project requirement, please ensure to refactor your code to ES6 syntax, which includes using arrow functions :+1:. This is important because ES6 provides several new features and syntax improvements that can make the code more concise and easier to read. Arrow functions, for instance, provide a shorter and more elegant syntax for writing functions and can make the code more readable and maintainable. Therefore, refactoring to ES6 syntax can improve the overall quality and maintainability of the code.